lightning/src/ln/channelmanager.rs

```diff
@@ -3234,6 +3238,7 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 										phantom_shared_secret,
 									},
 									value: amt_to_forward,
+									ticks: 0,
```

Comment:
I should probably call this `timer_ticks`.

Reply:
Yep, probably should :)